tdrs-backend/tdpservice/settings/common.py

```diff
@@ -110,6 +110,15 @@ class Common(Configuration):
     # Email Server
     EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
 
+    # Whether to use localstack in place of a live AWS S3 environment
+    USE_LOCALSTACK = bool(os.getenv("USE_LOCALSTACK", 0))
```

Comment:
Using the default here makes it so we don't have to set the variable in the deployed environments, where it will default to using the actual S3 environments.

Reply:
Makes sense! 👍 👍